File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/update-type-training-event-participants_param.ts


import { IUpdateTypeTrainingEventParticipants_Param } from './iupdate-type-training-event-participants_param';

export class UpdateTypeTrainingEventParticipants_Param implements IUpdateTypeTrainingEventParticipants_Param {
  
	public TrainingEventID: number = 0;
	public PersonIDs: number[] = null;
	public TrainingEventParticipantTypeID: number = 0;
	public RemovalReasonID?: number;
	public RemovalCauseID?: number;
	public DateCanceled?: Date;
  
}

File: GTTS/INL.TrainingService.Models/UpdateTypeTrainingEventParticipants_Param.cs
using System;

namespace INL.TrainingService.Models
{
    public class UpdateTypeTrainingEventParticipants_Param : IUpdateTypeTrainingEventParticipants_Param
    {
        public long TrainingEventID { get; set; }
        public long[] PersonIDs { get; set; }
        public int TrainingEventParticipantTypeID { get; set; }
        public long? RemovalReasonID { get; set; }
        public long? RemovalCauseID { get; set; }
        public DateTime? DateCanceled { get; set; }
    }
}
